### ISSUE TITLE
unable to install, can't run configure

### ISSUE THREAD

**Issue by revintec** [Vonng/pigsty#227]
using v1.5.1, latest version as of writing, on Debian. so it seems only CentOS is supported?

![image](https://user-images.githubusercontent.com/8952734/216031675-c202acae-5f12-4afe-8ee2-3721ce2b5eb9.png)
![image](https://user-images.githubusercontent.com/8952734/216031821-d6b9990a-ce04-4a78-a11b-0700b9146dbd.png)

**Comment by Vonng** [comment]
Yeah currently works on EL compatible distributions: RHEL, CentOS, RockyLinux, AlmaLinux, OracleLinux x EL 7,8,9...
